data/entities.json

```diff
@@ -563,7 +564,8 @@
       "yahoo.net",
       "yahooapis.com",
       "yimg.com",
-      "zenfs.com"
+      "zenfs.com",
+      "pixel.advertising.com"
```

Comment:
@soulgalore this is already being tracked under AOL / Oath / Verizon Media I believe, they're all the same company know but the branding was a bit different at one point.

Maybe we merge them all? but we should try to avoid splitting the `*.advertising.com` domain to two different entities

Reply:
I see, sorry let me have a look tonight. Strange though because I ran against the latest version and got a none match for that domain (testing against my top 100 list) (but I never actually checked against the json when I did the PR). I'll be back in a couple of hours.

Reply:
oooooh this actually broke in 0.3.0, I did a mass rename of categories and accidentally swept up advertising.com too :( 😞

Reply:
I'll push a fix

Reply:
Ah thanks for fixing it. Would love a 0.4.0 :)

Reply:
done!